notebook/static/notebook/js/actions.js

```diff
@@ -444,7 +451,7 @@ define(function(require){
                 return true;
             }
         },
-        'move-cursor-up-or-previous-cell':{
```

Comment:
I don't feel strongly about this - it was the consensus of the 4 of us that the `or-previous-cell` was logical enough that it could be assumed. Also, the `or-previous-cell` doesn't clearly express what is happening, which is more like "move cursor up in current or previous cell". More than willing to change this back though-even better would be to find a clear way of spelling it out.